cpp/include/cugraph/algorithms.hpp

```diff
@@ -1236,6 +1239,74 @@ void pagerank(raft::handle_t const& handle,
               bool has_initial_guess  = false,
               bool do_expensive_check = false);
 
+/**
+ * @brief Metadata about the execution of one of the centrality algorithms
+ */
+// FIXME:  This structure should be propagated to other algorithms that converge
+//   (eigenvector centrality, hits and katz centrality)
+//
+struct centrality_algorithm_metadata_t {
+  size_t number_of_iterations_;
+  bool converged_;
```

Comment:
Sorry for nitpicking, but better add `{}` at the end for more deterministic behaviors on error cases (users forgot to set these variables).

```
  size_t number_of_iterations_{};
  bool converged_{};
 ```

Reply:
Addressed in next push